filer/views.py

```diff
@@ -21,7 +22,11 @@ def selectfolder_param(request, separator="&"):
         return ""
 
 def current_site_param(request, separator="&"):
-    if 'current_site' in request.REQUEST:
-        return '%scurrent_site=%s' % (separator,
-                                      request.REQUEST['current_site'])
+    current_site = get_param_from_request(request, 'current_site')
+    if current_site:
+        return '%scurrent_site=%s' % (separator, current_site)
     return ""
+
+
+def get_param_from_request(request, param, default=None):
+    return request.POST.get(param) or request.GET.get(param) or default
```

Comment:
Yes, it is. I used the correct method where the usage is clear, but in most places we handle both POST and GET in the same method. I hoped that this change makes the issue obvious and future changes will fix it on a per view basis.

Do we want to make these changes with this PR ?